job-utilities/local-development/local_scheduler.py

```diff
@@ -0,0 +1,58 @@
+"""
```

Comment:
Why can't the local_scheduler and the lambda_function be the same file?  Do we need 2 that look like essentially the same thing?  At least at the beginning they look to be the same.

Reply:
There is no reason for the code in local_scheduler to be deployed anywhere, and if it was a part of lambda_function.py it would be zipped up and deployed
local_scheduler uses a python library to schedule events to send a request to local endpoints, the lambda_function takes input from an external scheduler to send off a request elsewhere